model/src/conditions/inline-condition-operators.js

```diff
@@ -92,3 +103,30 @@ function formatValue (fieldType, value) {
   }
   return `'${value}'`
 }
+
+export const absoluteDateOrTimeOperatorNames = Object.keys(absoluteDateTimeOperators)
+export const relativeDateOrTimeOperatorNames = Object.keys(relativeTimeOperators(dateTimeUnits))
+
+function absoluteDateTime (operator) {
+  return {
+    expression: (field, value) => {
+      if (value instanceof ConditionValue) {
+        return `${field.name} ${operator} '${value.toExpression()}'`
```

Comment:
should toExpression be a getter? `value.expression`

Reply:
Possibly, this feels like perhaps a nice to have that we can address in a later pr?